opencti-platform/opencti-graphql/src/domain/stixObjectOrStixRelationship.ts

```diff
@@ -31,8 +33,16 @@ export const stixObjectOrRelationshipAddRefRelation = async (
   if (!isStixRefRelationship(finalInput.relationship_type)) {
     throw FunctionalError(`Only ${ABSTRACT_STIX_REF_RELATIONSHIP} can be added through this method.`);
   }
-  const relation = await createRelation(context, user, finalInput);
-  await notify(BUS_TOPICS[type as BusTopicsKeyType].EDIT_TOPIC, relation.from, user);
+  // Create relation
+  const to : StoreMarkingDefinition = await findById(context, user, input.toId);
+  const fromRole = `${type}_from`;
+  const fromType = stixObjectOrRelationship.entity_type;
+  const fieldName = schemaRelationsRefDefinition.convertDatabaseNameToInputName(fromType, input.relationship_type);
+  const patch = { [fieldName as string]: [input.toId] };
+  const operations = { [fieldName as string]: UPDATE_OPERATION_ADD };
+  const { element } = await patchAttribute(context, user, stixObjectOrRelationshipId, type, patch, { operations });
+  const relation = { ...element, from: stixObjectOrRelationship, fromId: stixObjectOrRelationshipId, fromRole, fromType, to };
+  await notify(BUS_TOPICS[type as BusTopicsKeyType].EDIT_TOPIC, relation, user);
   return relation;
 };
 export const stixObjectOrRelationshipAddRefRelations = async (
```

Comment:
Should we call stixObjectOrRelationshipAddRefRelation instead of createRelations in stixObjectOrRelationshipAddRefRelations method ?

Reply:
Yes, it will be a to-do for later